### PR TITLE
Cw 682 integrate stealth ex exchange provider

### PR DESCRIPTION
**Pull request by Serhii-Borodenko**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [x] Double check modified code and verify it with the feature/task requirements
- [x] Format code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

File: lib/exchange/provider/stealth_ex_exchange_provider.dart [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  Future<Map<String, dynamic>> getEstimatedExchangeAmount(
+      {required CryptoCurrency from,
+      required CryptoCurrency to,
+      required double amount,
+      required bool isFixedRateMode}) async {
+    final headers = {'Authorization': apiKey, 'Content-Type': 'application/json'};
+
+    final body = {
+      'route': {
+        'from': {'symbol': from.name, 'network': 'mainnet'},
```
Comment (Blazebrain) [inline lib/exchange/provider/stealth_ex_exchange_provider.dart:225]: Are we sure it's only the `mainnet` network we'll be dealing with?

File: lib/exchange/provider/stealth_ex_exchange_provider.dart [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  @override
+  Future<Limits> fetchLimits(
+      {required CryptoCurrency from,
+      required CryptoCurrency to,
+      required bool isFixedRateMode}) async {
+    final headers = {'Authorization': apiKey, 'Content-Type': 'application/json'};
+    final body = {
+      'route': {
+        'from': {'symbol': isFixedRateMode ? to.name : from.name, 'network': 'mainnet'},
+        'to': {'symbol': isFixedRateMode ? from.name : to.name, 'network': 'mainnet'}
```
Comment (Blazebrain) [inline lib/exchange/provider/stealth_ex_exchange_provider.dart:54]: Same question about the network, will we only be dealing with `mainnet`?

**Review by Blazebrain — COMMENTED** [review]
Great work on this 🚀

Once the comments are answered, its good to go from my end

File: lib/exchange/provider/stealth_ex_exchange_provider.dart (hunk 2 above)
Comment (Serhii-Borodenko) [reply lib/exchange/provider/stealth_ex_exchange_provider.dart]: fixed, thanks for pointing that out.

File: lib/exchange/provider/stealth_ex_exchange_provider.dart
Comment (Habtamu49) [inline lib/exchange/provider/stealth_ex_exchange_provider.dart:1]: lib/exchange/provider/stealth_ex_exchange_provider.dart

Comment (Habtamu49) [reply lib/exchange/provider/stealth_ex_exchange_provider.dart:1]: lib/exchange/exchange_provider_description.dart

Comment (Habtamu49) [reply lib/exchange/provider/stealth_ex_exchange_provider.dart:1]: Ok

**Review by Habtamu49 — COMMENTED** [review]
lib/view_model/exchange/exchange_view_model.dart

**Pull request merged**